### ISSUE TITLE
Hide user id & password input if local session already exists

### ISSUE THREAD

**Issue by aaravlu** [project-robius/robrix#286]

**Comment by kevinaboos** [comment]
yeah, this is on my to-do list too. Instead of fully hiding the input fields, what I'd like to do is send an action to the LoginScreen with the username being logged into such that we can display that username to the user, and then gray out the fields and the "Login" button while the auto-login procedure is underway.

I actually already built this concept into the `LoginAction` enum here: https://github.com/project-robius/robrix/blob/6b84169b38810d7dfc95c2ce7a66573a379badc5/src/login/login_screen.rs#L501-L507

it just hasn't been implemented by the LoginScreen itself.

**Comment by kevinaboos** [comment]
Upon further thought, I think the correct way to do this is to:
* re-do the status label by removing it in favor of a small pop-up modal that shows the status
* A pop-up modal itself will be overlaid atop the login content, such that the underlying TextInputs and buttons are blocked. This makes it easier to prevent multiple simultaneous login attempts than trying to gray-out or disable the TextInputs/buttons.
* The `LoginAction::AutofillInfo` can likely just be removed in favor a generic mechanism (a new variant in `LoginAction`) to show that login popup modal.

I plan to work on this shortly.

**Comment by kevinaboos** [comment]
> Upon further thought, I think the correct way to do this is to:
> 
> * re-do the status label by removing it in favor of a small pop-up modal that shows the status
> * A pop-up modal itself will be overlaid atop the login content, such that the underlying TextInputs and buttons are blocked. This makes it easier to prevent multiple simultaneous login attempts than trying to gray-out or disable the TextInputs/buttons.
> * The `LoginAction::AutofillInfo` can likely just be removed in favor a generic mechanism (a new variant in `LoginAction`) to show that login popup modal.
>   
>   * We can use that modal for many purposes: status/loading messages, as an automatic pop-up when logging in via CLI arguments or when restoring a previous session, etc.

I implemented all of this as part of a complete redesign of the LoginScreen in #320.
